### ISSUE TITLE
ModuleNotFoundError: No module named 'art.defences.detector.poisoning'

### ISSUE THREAD

**Issue by jiep** [Trusted-AI/adversarial-robustness-toolbox#632]
**Describe the bug**
Module not found after importing any defense from module `art.defences.detector.poisoning`.

**To Reproduce**
1. Install ART: `pip install adversarial-robustness-toolbox==1.4.0`
2. Run Python: `python`
3. Import any defense from module `art.defences.detector.poisoning`: 

```python
>>> from art.defences.detector.poisoning import ProvenanceDefense
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'art.defences.detector.poisoning'
>>> from art.defences.detector.poisoning import RONIDefense
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'art.defences.detector.poisoning'
>>> from art.defences.detector.poisoning import SpectralSignatureDefense
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'art.defences.detector.poisoning'
>>> from art.defences.detector.poisoning import ActivationDefence
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'art.defences.detector.poisoning'
```

**Expected behavior**
Module `art.defences.detector.poisoning` should exist according to [documentation](https://adversarial-robustness-toolbox.readthedocs.io/en/latest/modules/defences/detector_poisoning.html#module-art.defences.detector.poisoning).

**System information (please complete the following information):**
 - OS: `Debian GNU/Linux 10 (buster)`. Same error using Google Colab.
 - Python version: `3.7.9`
 - ART version or commit number: `1.4.0`
 - TensorFlow / Keras / PyTorch / MXNet version: 

```
tensorflow==2.3.1
Keras==2.4.3
torch==1.6.0
```

**Comment by beat-buesser** [comment]
Hi @jiep Thank you for raising this issue. Could you please confirm that you have installed ART 1.4.0?

Following your reproduction steps I was not able to reproduce the issue with ART 1.4.0.

**Comment by jiep** [comment]
Thanks for the quick answer.

I confirm ART 1.4.0 is installed. I created a notebook to reproduce the issue. [Error.ipynb.zip](https://github.com/Trusted-AI/adversarial-robustness-toolbox/files/5311528/Error.ipynb.zip). 

> Note: The file is a `zip` because GitHub doesn't allow to upload `.ipynb` files.

The output I get after running the notebook on Google Colab is the following:

<img width="724" alt="Captura de pantalla 2020-10-01 a las 13 15 52" src="https://user-images.githubusercontent.com/414463/94802975-db365880-03e8-11eb-8052-86270e5ea74b.png">

**Comment by beat-buesser** [comment]
Thank you very much, we can now see the problem. We will fix it with tomorrows 1.4.1 release.

**Comment by beat-buesser** [comment]
You should be able to continue today with `art.defences.detector.poison` instead of `art.defences.detector.poisoning` if you install 1.4.0 from PyPI.
